packages/client/tests/tests-utils.tsx

```diff
@@ -143,9 +143,20 @@ export function getFakeUsersList({
 }
 
 export function extractTrackIDFromCardContainerTestID(testID: string): string {
-    return testID.replace('-track-card-container', '');
+    return testID
+        .replace('-track-card-container', '')
+        .replace('mpe-', '')
+        .replace('mtv-', '');
```

Comment:
I would expect more defensive code here.
When I want to extract some data, I prefer to focus on what I want instead of what I don't want.

I would use a RegExp to extract the `trackID`, by hardly expecting the pattern. If the pattern changes, we would like other developers to check manually that this function still does what we want.

```suggestion
    const result = testID.match(/(?:mpe|mtv)-(?<trackID>.+)-track-card-container/);
    invariant(result !== null, 'Pattern was not recognized');
    
    const trackID = result.groups.trackID;
    invariant(typeof trackID === 'string', 'TrackID named group has not been captured');
    
    return trackID;
```

Reply:
You're right this clearly more readable for everyone.
Great suggestion thanks !
The testID prefix can be either 'mtv' or 'mpe' but also undefined
In this way we should add a third occurence the first regex query
```regex
(?:mpe-|mtv-|)(?<trackID>.+)-track-card-container
```
Pushing a related commit soon